packages/tooltip/src/tooltipUtils.tsx

```diff
@@ -63,15 +61,15 @@ export function trianglePosition(
         notchStyleObj.left = `${notchOverlap}px`;
 
         // right align is .8px off, factoring that in here to correct offset
-        containerStyleObj.left = `calc(100% - 1.8px)`;
+        containerStyleObj.right = `-${containerSize}px`;
```

Comment:
Looks like we're no longer factoring in that .8px so we can rm the comment above

Reply:
Done!